app/portainer/views/jobs/create/createJobController.js

```diff
@@ -0,0 +1,71 @@
+angular.module('portainer.app')
+  .controller('CreateJobController', ['$scope', '$state', 'JobService', 'Authentication', 'Notifications', 'EndpointProvider', 'StateManager',
+    function ($scope, $state, JobService, Authentication, Notifications, EndpointProvider, StateManager) {
+
+      $scope.formValues = {
+        Image: 'ubuntu:latest',
+        JobFileContent: '',
+        JobFile: null
+      };
+
+      $scope.state = {
+        Method: 'editor',
+        formValidationError: '',
+        actionInProgress: false
+      };
+
+      function createJob(image, method) {
+        var endpointId = EndpointProvider.endpointID();
+
+        if (method === 'editor') {
```

Comment:
use `switch`?

Reply:
As we have only 2 values and no default, not sure about the switch here.